LICENSE

```diff
@@ -1,661 +1,14 @@
-                    GNU AFFERO GENERAL PUBLIC LICENSE
-                       Version 3, 19 November 2007
+BSD 3-Clause License
 
- Copyright (C) 2007 Free Software Foundation, Inc. <http://fsf.org/>
- Everyone is permitted to copy and distribute verbatim copies
- of this license document, but changing it is not allowed.
+Copyright (c) 2020, Robert Gieseke, Zebedee Nicholls, Jared Lewis, Sven Willner, Matthias Mengel
+All rights reserved.
```

Comment:
```suggestion
```
I'd remove this as it makes unclear and is not necessary.
https://opensource.stackexchange.com/questions/2121/mit-license-and-all-rights-reserved

Reply:
Resolved in 1d12d629